### PR TITLE
Added Todays Tasks

### PR DESCRIPTION
**Pull request by Mhorvilleur13**
# Context

I added a page for tasks that are today. If there are no tasks for today, the page will say so

# Tests
Same old 

Screen Shot 2022-03-29 at 12 50 19 PM

# Screenshots

Attach screenshots, before and after
Screen Shot 2022-03-29 at 12 51 00 PM

### CODE REVIEW

File: src/state/atoms.ts [hunk 1]
```diff
@@ -1,17 +1,15 @@
 import dayjs from "dayjs";
+import isToday from "dayjs/plugin/isToday";
 import { atom, selector, useRecoilState } from "recoil";
 import { Task } from "../types/task";
 
+dayjs.extend(isToday);
```
Comment (guzzur) [inline src/state/atoms.ts:6]: I would add a file in services (e.g. timeUtils.ts) and export extended dayjs object from there, so you don't need to do it everywhere you want to use it.

**Review by guzzur — COMMENTED** [review]
Amazing PR. Only one nittiest nit.

File: src/state/atoms.ts (hunk 1 above)
Comment (Mhorvilleur13) [reply src/state/atoms.ts:6]: Once I export the object, how can I call is in another file? I am having trouble with this export.

Comment (Mhorvilleur13) [reply src/state/atoms.ts:6]: How do I export dayjs.extend(isToday)? I have tried several ways and none work.

**Pull request merged**